crates/metal-mir/src/types/function.rs

```diff
@@ -2,8 +2,12 @@ use super::{visibility::Visibility, Type};
 
 #[derive(Debug, Clone)]
 pub struct FunctionSignature {
+    /// Function name
     pub name: String,
+    /// Represents the type returned by this function
     pub return_type: Type,
+    /// Represents the arguments this function takes
     pub arguments: Vec<Type>,
```

Comment:
While args and params are often used interchangeably in day-to-day speech, they still mean different things. Arguments are values passed when calling a function, while parameters are arguments that a function accepts. To help highlight the difference between the two, "inputs" is sometimes used instead of "parameters", and is what AST does already. Therefore, it think it will be better to rename this to `inputs` both for the above reason and for consistency with AST.

Reply:
Sure!